apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    IDetailQueryResult,
+    ITranscribedAudioViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Link } from 'react-router-dom';
+import { routes } from '../../../app/routes/routes';
+import './transcribed-audio-detail.thumbnail.presenter.css';
+
+export const TranscribedAudioDetailThumbnailPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const location = `/${routes.resources
+        .ofType(ResourceType.transcribedAudio)
+        .detail(transcribedAudioItem.id)}`;
+
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
+
+    return (
+        <div className="detail-thumbnail-container">
+            <Link to={location}>
+                <div
+                    className="detail-thumbnail-meta-container"
+                    title="View Connected Transcribed Audio"
+                >
+                    <strong>Transcribed Audio ID:</strong> {transcribedAudioItem.id}
```

Comment:
What is the significanace of `<strong>...</strong></br>`? It seems to be a formatting rule we need a single source of truth for in a utils \ shared dir.

Reply:
I'm just using this for visual formatting.  It could also just be `<b>`, or it could be a `<span>` with a class name that is something like `property-label`